docs/tables/aws_audit_manager_control.md

```diff
@@ -0,0 +1,31 @@
+# Table: aws_audit_manager_control
+
+AWS Audit Manager helps to continuously audit AWS usage to simplify how you assess risk and compliance with regulations and industry standards. With Audit Manager, it is easy to assess if policies, procedures, and activities – also known as controls – are operating effectively.
+
+## Examples
+
+### Basic info
+
+```sql
+select
+  name,
+  id,
+  description,
+  type
+from
+  aws_audit_manager_control;
+```
+
+
+### List custom audit manager controls
```

Comment:
@rajeshbal65 @Subhajit97 is it going to give me list of `Custom frameworks`
![image](https://user-images.githubusercontent.com/26873346/119790551-a6d05900-bef1-11eb-9f69-803f4ff21c4b.png)

Reply:
@rajmohanty17 We have a separate table (aws_auditmanager_framework) to list the `assessment framework`.
This table will list auditmanager controls (i.e. both Custom and Standard).